contracts/reflect/src/contract.rs

```diff
@@ -106,10 +99,9 @@ pub fn try_change_owner(
         state.owner = api.addr_validate(&new_owner)?;
         Ok(state)
     })?;
-    Ok(Response {
-        attributes: vec![attr("action", "change_owner"), attr("owner", new_owner)],
-        ..Response::default()
-    })
+    Ok(Response::new()
+        .add_attribute(("action", "change_owner"))
```

Comment:
`add_attributes` wouldn't work here as they are different value types, right?

Reply:
Yes, I think that's going to be true for most of these cases unfortunately.